src/pyodide.js

```diff
@@ -178,6 +178,7 @@ var languagePluginLoader = new Promise((resolve, reject) => {
         }
         script.onerror = (e) => { reject(e); };
         document.body.appendChild(script);
+        window.onerror = (e) => { reject(e); };
```

Comment:
Why is this change needed?

Reply:
@mdboom I just checked, I think we can eliminate this perhaps. Thanks for pointing out!

Reply:
@mdboom I checked again. It turns out this is necessary so as to reject the promise when the exception occurs. If not done, this results in a ``TimeOutException`` as it tries to load the package even after the exception is thrown. I was probably testing out with a previous version of ``pyodide.js``, so please ignore the above response perhaps.

Reply:
Ok.  Makes sense.  It should probably come before the `document.body.appendChild` so there isn't a race condition if the error happens early.